clientv2/client.go

```diff
@@ -134,6 +136,157 @@ func (c *Client) Post(ctx context.Context, operationName, query string, respData
 	return f(ctx, req, gqlInfo, respData, c.do)
 }
 
+type MultipartFile struct {
+	File  graphql.Upload
+	Index int
+}
+
+type MultipartFilesGroup struct {
+	Files      []MultipartFile
+	IsMultiple bool
+}
+
+type FormField struct {
+	Name  string
+	Value interface{}
+}
+
+// PostMultipart
+// support send multipart form with files https://gqlgen.com/reference/file-upload/ https://github.com/jaydenseric/graphql-multipart-request-spec
+func (c *Client) PostMultipart(ctx context.Context, operationName, query string, respData interface{}, vars map[string]interface{}, interceptors ...RequestInterceptor) error {
```

Comment:
Why is the reason for separating it from Post?

Reply:
1) Keep Post method more simply. 
2) if operation has graphql.Upload|*[]graphql.Upload argument need use PostMultipart else use written Post()

Reply:
done